extensions/wikia/CategorySelect/CategorySelectController.class.php

```diff
@@ -85,7 +85,7 @@ public function categories() {
 				}
 				$text = $this->wg->ContLang->convertHtml( $title->getText() );
 				$data[ $name ] = array(
-					'link' => Linker::link( $title, $text ),
+					'link' => Linker::link( $title, $text , [], [], [ 'known' ] ),
```

Comment:
I'm not sure if this behavior is correct—the category should be redlinked if the page doesn't exist

Reply:
At the moment of changing categories on the bottom part of the page there is lag. The moment the request is made those new categories don't exist. As far as I can tell categories are always created when someone adds them to an article

Reply:
You can add a nonexistent category to the article and the category page can be visited and will show the article as a member, but that doesn't mean the category page itself exists as a wiki page—it must be created manually.